lib/ez/settings/backend/active_record.rb

```diff
@@ -0,0 +1,31 @@
+module Ez::Settings
+  module Backend
+    class ActiveRecord
+      def read
```

Comment:
please, add to TODOs: "[ ] - Backend#read method should receive app, group & key attributes and avoid loading all in case of SQL adapter"